pandas/tests/generic/test_duplicate_labels.py

```diff
@@ -94,7 +94,6 @@ def test_preserve_getitem(self):
         assert df.loc[[0]].flags.allows_duplicate_labels is False
         assert df.loc[0, ["A"]].flags.allows_duplicate_labels is False
 
-    @pytest.mark.xfail(reason="Unclear behavior.")
```

Comment:
@TomAugspurger IIRC you were the driving force behind `flags`.  do you expect to finish these off in the not-too-distant future?

Reply:
No plans right now.